qt/widgets/common/src/AlgorithmHistoryWindow.cpp

```diff
@@ -509,6 +515,22 @@ void AlgorithmHistoryWindow::doUnroll(const std::vector<int> &unrollIndicies) {
 
 void AlgorithmHistoryWindow::doRoll(int index) { m_view->roll(index); }
 
+void AlgorithmHistoryWindow::unrollAll(int state) {
+  // Iterate all items in tree which have child algorithms to be unrolled
+  QTreeWidgetItemIterator it(m_Historytree,
+                             QTreeWidgetItemIterator::HasChildren);
+
+  auto head = m_Historytree->headerItem();
```

Comment:
I'm suprised the CI didn't warn on this. I have a compiler warning now that the debugging statement has been removed:
``` 
[6/39] Building CXX object qt/widgets/common/CMakeFiles/MantidQtWidgetsCommonQt5.dir/src/AlgorithmHistoryWindow.cpp.o
../../qt/widgets/common/src/AlgorithmHistoryWindow.cpp: In member function ‘void AlgorithmHistoryWindow::unrollAll(int)’:
../../qt/widgets/common/src/AlgorithmHistoryWindow.cpp:523:8: warning: unused variable ‘head’ [-Wunused-variable]
   auto head = m_Historytree->headerItem();
```